props/saber_sabersense_buttons.h

```diff
@@ -465,6 +490,61 @@ class SabersenseButtons : public PROP_INHERIT_PREFIX PropBase {
     }
 #endif
   }
+  
+  // PRESET SKIPPING
+  // Skip Value definable.
+  // One Button System - Up, Horizontal, Down.
+  bool OneBtnMultiFontSkip(int skip_value) {
+      int count = 0;
+      CurrentPreset tmp;
+      for (int i = 0; ; i++) {
```

Comment:
There is an easier way to count presets:

You can see it here:

https://github.com/profezzorn/ProffieOS/blob/3c60a823d4da314134870e2e9c91966132b7b91d/props/saber_fett263_buttons.h#L2988